package.json

```diff
@@ -8,9 +8,164 @@
+    "start:content-watcher": "nest start content-watcher",
+    "start:content-watcher:watch": "nest start content-watcher --watch",
+    "start:content-watcher:prod": "node dist/apps/content-watcher/main.js",
+    "start:content-watcher:dev": "dotenvx run -f .env.content-watcher -- nest start content-watcher --watch",
+    "start:content-watcher:debug": "dotenvx run -f .env.content-watcher -- nest start content-watcher --debug --watch",
+    "start:graph-api": "nest start graph-api",
+    "start:graph-api:watch": "nest start graph-api --watch",
+    "start:graph-api:prod": "node dist/apps/graph-api/main.js",
+    "start:graph-api:dev": "dotenvx run -f .env.graph -- nest start graph-api --watch",
+    "start:graph-api:debug": "dotenvx run -f .env.graph -- nest start graph-api --debug --watch",
+    "start:graph-worker": "nest start graph-worker",
+    "start:graph-worker:watch": "nest start graph-worker --watch",
+    "start:graph-worker:prod": "node dist/apps/graph-worker/main.js",
+    "start:graph-worker:dev": "dotenvx run -f .env.graph -- nest start graph-worker --watch",
+    "start:graph-worker:debug": "dotenvx run -f .env.graph -- nest start graph-worker --debug --watch",
+    "generate:metadata:account": "npx ts-node -r tsconfig-paths/register apps/account-api/src/generate-metadata.ts",
+    "generate:metadata:content-publishing": "npx ts-node -r tsconfig-paths/register apps/content-publishing-api/src/generate-metadata.ts",
+    "generate:metadata:content-watcher": "npx ts-node -r tsconfig-paths/register apps/content-watcher/src/generate-metadata.ts",
+    "generate:metadata:graph": "npx ts-node -r tsconfig-paths/register apps/graph-api/src/generate-metadata.ts",
+    "generate:metadata": "npm run generate:metadata:account ; npm run generate:metadata:content-publishing ; npm run generate:metadata:content-watcher ; npm run generate:metadata:graph",
+    "generate:openapi:account": "nest start account --entryFile build-openapi",
+    "generate:openapi:content-publishing": "nest start content-publishing-api --entryFile build-openapi",
+    "generate:openapi:content-watcher": "nest start content-watcher --entryFile build-openapi",
+    "generate:openapi:graph": "nest start graph-api --entryFile build-openapi",
+    "generate:openapi": "npm run generate:openapi:account ; npm run generate:openapi:content-publishing ; npm run generate:openapi:content-watcher ; npm run generate:openapi:graph",
+    "generate:types:graph": "npx openapi-client-axios-typegen openapi-specs/graph-webhooks.openapi.yaml > libs/graph-lib/src/types/webhook-types.d.ts ; prettier --write libs/graph-lib/src/types/webhook-types.d.ts",
+    "generate:types:content-watcher": "npx @hey-api/openapi-ts -i openapi-specs/content-announcement.openapi.json -o libs/content-watcher-lib/src/types/content-announcement ; prettier --write libs/content-watcher-lib/src/types/content-announcement",
```

Comment:
Why are we using different typegen libraries?

Reply:
This came directly from the different microservices. The standard OpenAPI 'typegen' doesn't support enums properly, and I wasn't sure if we want to standardize on a forked package, so I left it this way for now.